crates/bevy_ui/src/node_bundles.rs

```diff
@@ -20,7 +20,7 @@ use bevy_transform::prelude::{GlobalTransform, Transform};
 pub struct NodeBundle {
     /// Describes the logical size of the node
     pub node: Node,
-    /// Describes the style including flexbox settings
+    /// Describes the style including layout styles
```

Comment:
this raise the question, what other styles are there?

Reply:
Things like colors (background color, border color), text styles, etc. In general: things that affect painting/drawing rather than layout.

Reply:
could you improve those doc with those informations?

Reply:
Ok, it now says:

> Styles which control the layout (size and position) of the node and it's children
> In some cases these styles also affect how the node drawn/painted.

I think it's mostly `overflow` which affects painting as it control clipping.